components/closebutton/stories/closebutton.stories.js

```diff
@@ -30,6 +46,29 @@ export default {
 export const Default = CloseButtonGroup.bind({});
 Default.args = {};
 
+/**
+ * Close button provides a "large" icon size option, for displaying a larger cross icon for each component size.
+ * When using this option, the following UI icons should be used:
+ * 
+ * | Close button class name         | UI icon class name          |
+ * | ------------------------------- | --------------------------- |
+ * | `.spectrum-CloseButton--sizeS`  | `.spectrum-UIIcon-Cross200` |
+ * | `.spectrum-CloseButton--sizeM`  | `.spectrum-UIIcon-Cross300` |
+ * | `.spectrum-CloseButton--sizeL`  | `.spectrum-UIIcon-Cross400` |
+ * | `.spectrum-CloseButton--sizeXL` | `.spectrum-UIIcon-Cross500` |
+ */
+export const SizingLargeIcons = (args, context) => Sizes({
```

Comment:
Note: I included this story in VRTs, since it can't currently be covered by the `Variants()` function. It might be nice in the future if that function could allow its use of `Sizes()` to iterate over additional test data (in this case, `iconSize: "large"`).